solutions/usaco-1228

```diff
@@ -0,0 +1,73 @@
+---
+id: usaco-1228
+source: USACO Bronze 2022 Open
+title: Counting Liars
+author: Chongtian Ma
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_prob2_bronze_open22.html)
+
+## Solution 1
+
+See the official analysis.
+
+## Solution 2 - Sorting
```

Comment:
```suggestion
## Alternate Explanation
```

Reply:
well its not really alternate because the official doesn't mention the observation lol.
its more like "additional" i guess?

Reply:
hm
but your algorithm is n log n while official sol is n^2
best use this